tvb/simulator/plot/utils/log_error_utils.py

```diff
@@ -0,0 +1,73 @@
+# -*- coding: utf-8 -*-
```

Comment:
This contradicts logging in tvb core modules. This I would not merge

Reply:
This has been reviewed/rewritten and it can be viewed in a later commit.